src/ActionList/ActionList.stories.tsx

```diff
@@ -241,35 +241,17 @@ LinkItemPlayground.argTypes = {
     type: 'string',
   },
 }
-
-export const GroupPlayground: Story<ActionListGroupProps> = args => (
-  <ActionList>
-    <ActionList.Group {...args}>
-      <ActionList.Item>Item 1</ActionList.Item>
-      <ActionList.Item>Item 2</ActionList.Item>
-    </ActionList.Group>
+export const HeadingPlayground: Story<ActionListProps> = args => (
+  <ActionList headingProps={{title: 'Group title'}} {...args}>
+    <ActionList.Item>Copy link</ActionList.Item>
+    <ActionList.Item>Quote reply</ActionList.Item>
+    <ActionList.Item>Edit comment</ActionList.Item>
   </ActionList>
 )
-GroupPlayground.argTypes = {
-  variant: {
-    control: {
-      type: 'radio',
-    },
-    options: ['subtle', 'filled'],
-  },
-  role: {
-    type: 'string',
-  },
-  title: {
-    type: 'string',
-  },
-  auxiliaryText: {
-    type: 'string',
-  },
-}
-GroupPlayground.args = {
-  variant: 'subtle',
-  role: 'listbox',
-  title: 'Group title',
-  auxiliaryText: '',
+
+HeadingPlayground.args = {
```

Comment:
I think you may need to set the `argTypes` here so we can toggle between `subtle` and `filled` variants. I wonder if you want to delete the contents of this list just to render the title, and then focus all the controls around options that the `Heading` component offers?

Right now I'm just seeing list controls
![image](https://user-images.githubusercontent.com/18661030/224436848-bdf51a79-c723-42e3-ad77-6ad243ec3c16.png)